README.md

```diff
@@ -91,12 +95,13 @@ That's it! Your application should be running. To proceed with your development,
 If you encounter any problems, try the Troubleshooting section.
 
 * explore `config/env/development.js` for development environment configuration options
+* it is possible to
```

Comment:
Oh, I just forgot to actually end that text... I wanted to refer to @mleanos 's comment about doing 'npm run gulp <task>' - checkout the updated PR.